### ISSUE TITLE
fix(form): independence from validator broken

### ISSUE THREAD

**Issue by ayoayco** [ayoayco/astro-reactive-library#136]
After implementing server-side rendered validation errors, the validator package was needed in `form-control.ts`

This breaks the independence of our packages.

Find a way such that validator is not a dependency of the form package.

**Comment by ayoayco** [comment]
Found solution here: https://stackoverflow.com/questions/15302618/node-js-check-if-module-is-installed-without-actually-requiring-it

**Comment by Lalit3716** [comment]
Also, I think It would be nice if we could show users a warning in the terminal if they used validators in their form control but forgot to install the validator package!
Something like:

"Warning: Install @astro-reactive/validator to work with validators"?

**Comment by ayoayco** [comment]
Good idea, though we should think about logging in another issue. Maybe create a logger in the common package.
